rllib/env/multi_agent_env.py

```diff
@@ -405,10 +405,14 @@ def action_space_sample(self,
 
         @override(MultiAgentEnv)
         def action_space_contains(self, x: MultiAgentDict) -> bool:
+            if not isinstance(x, dict):
+                return False
```

Comment:
should this raise?

Reply:
also do we need to assume something stronger, ie that it is a MultiAgentDict?

Reply:
1. I don't think its should raise, because an argument could be made that any element passed to the contains function that is not apart of the observation space (no matter how incorrect the element is) should just be considered **not** in the observation space, and therefore we should just return False. We could raise a warning if the element is not properly formatted, but only inside the environment checking module in order to keep the overhead of functions like this low.
2. The MultiAgentDict is established as a type for documentation. I don't think you can actually instantiate a MultiAgentDict type. Its just a dict, where the structure of the dict has been pre-agreed upon.

Reply:
1) makes sense. 2) I see; can you check for that structure explicitly? That would make this check more precise, right?

Reply:
I've added checks for the specific structure in a followup pr for BaseEnv Checking -- The same components are used in bother checkers